ios/PolyPodApp/FeatureContainer/FeatureContainerView.swift

```diff
@@ -41,6 +41,66 @@ struct FeatureContainerView: UIViewRepresentable {
     }
 }
 
+class FeatureFileHandler: UIViewController, WKURLSchemeHandler {
+    private var feature: Feature? = nil
+    func setFeature(feature: Feature) {
+        self.feature = feature
+    }
+    
+    func mimeTypeFromExt(ext: String) -> String {
+        switch ext {
+        case "html":
+            return "text/html"
+        case "js":
+            return "application/javascript"
+        case "css":
+            return "text/css"
+        case "svg":
+            return "image/svg+xml"
+        case "json":
+            return "application/json"
+        default:
```

Comment:
Hum. Not octetstream? (or suchlike)

Reply:
Is this actually necessary? I'd rather not bother with setting the correct mime type for resources requested by the feature (or, if I get my way, `RemoteClientPod.readFile`). If we need that logic for some reason, we could always implement it there (since we know the extension all the same), not at the HTTP response level. That way we don't have to duplicate the logic for multiple platforms either.